mmengine/_strategy/deepspeed.py

```diff
@@ -311,8 +311,8 @@ def __init__(
         self.config['steps_per_print'] = steps_per_print
         self._inputs_to_half = inputs_to_half
         assert (exclude_frozen_parameters is None or
-                digit_version(deepspeed.__version__) >= digit_version('0.10.1')
-                ), ('DeepSpeed >= 0.10.1 is required to enable '
+                digit_version(deepspeed.__version__) >= digit_version('0.13.2')
```

Comment:
Why do we need to change this version? Does `deepspeed>=0.10.1 < 0.13.2` not support this parameter?

Reply:
@zhouzaida  Hi!

Starting from v0.10.1, `save_checkpoint` and `module_state_dict` support the `exclude_frozen_parameters`.
Starting from v0.13.2, `_zero3_consolidated_16bit_state_dict` supports the `exclude_frozen_parameters`.

I change the version to ensure the code is clean. Do you have better suggestions for this modification?